### PR TITLE
Feature/pastelstatistics transaction fee chart

### PR DESCRIPTION
**Pull request by Dinisueldanov**
### Summary
- added transaction fee chart.
- added tests for this chart.
### Screenshot
statistics-transaction-fee

### CODE REVIEW

File: src/features/pastelStatistics/__tests__/pastelLib.test.ts [hunk 1]
```diff
@@ -254,6 +258,84 @@ describe('managePastelDatabase', () => {
       dataX: [],
     })
   })
-})
 
-export {}
+  test('transformNetTotals function works correctly', async () => {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/__tests__/pastelLib.test.ts:262]: I think this test doesn't need for this PR. Can we remove it from this PR?

File: src/features/pastelStatistics/__tests__/pastelLib.test.ts [hunk 2]
```diff
@@ -254,6 +258,84 @@ describe('managePastelDatabase', () => {
...
+      dataX: [
+        '2021-05-20T13:42:13.277Z UTC (MockDate: GMT-0700)',
+        '2021-05-20T13:42:13.357Z UTC (MockDate: GMT-0700)',
+      ],
+      dataY1: [123434, 147102],
+      dataY2: [21296, 27015],
+    })
+  })
+
+  test('transformMempoolInfo function works correctly', async () => {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/__tests__/pastelLib.test.ts:282]: I think this test doesn't need for this PR. Can we remove it from this PR?

File: src/features/pastelStatistics/__tests__/pastelLib.test.ts [hunk 3]
```diff
@@ -254,6 +258,84 @@ describe('managePastelDatabase', () => {
...
+    expect(result).toEqual({
+      dataX: [
+        '2021-05-20T13:42:13.277Z UTC (MockDate: GMT-0700)',
+        '2021-05-20T13:42:13.357Z UTC (MockDate: GMT-0700)',
+      ],
+      dataY: [1.344, 1.56],
+    })
+  })
+
+  test('transformBlockSizeInfo function works correctly', async () => {
```
Comment (ngvtuan) [inline src/features/pastelStatistics/__tests__/pastelLib.test.ts:304]: I think this test doesn't need for this PR. Can we remove it from this PR?

**Review by ngvtuan — COMMENTED** [review]
https://user-images.githubusercontent.com/10944387/121635895-3e68a680-cab1-11eb-94c9-3b4cc7c4cfdf.mov

The chart does not show if we remove the `pasteldb.sqlite` file.

File: src/features/pastelDB/pastelDBLib.ts [hunk 4]
```diff
@@ -1007,3 +1008,35 @@ export function insertBlockChainInfo(
     pastelDB.exec(insertBlockChainInfoQuery, values)
   }
 }
+
+export function getTransactionAverageFeeDataFromDB(
```
[the remark this answers is not in the document]
Comment (Dinisueldanov) [reply src/features/pastelDB/pastelDBLib.ts:1012]: Thanks for your comments.
This function did not use in this pr, removed this here.

**Review by ngvtuan — DISMISSED** [review]

**Review by ngvtuan — APPROVED** [review]

**Review by sanjanadhikari1 — APPROVED** [review]
Looks good for me

File: src/features/pastelStatistics/components/averageblocksizeovertime/AverageBlockSizeOvertime.tsx [hunk 5]
```diff
@@ -0,0 +1,117 @@
...
+  const [ticker, setTicker] = useState<NodeJS.Timeout>()
+  const [
+    transformLineChartData,
+    setTransformLineChartData,
+  ] = useState<TLineChartData>()
+
+  useEffect(() => {
+    const loadLineChartData = async () => {
+      const pasteldb = await PastelDB.getDatabaseInstance()
+      const result = getFilteredDataFromDBByPeriod(
```
[the remark this answers is not in the document]
Comment (Dinisueldanov) [reply src/features/pastelStatistics/components/averageblocksizeovertime/AverageBlockSizeOvertime.tsx]: Thanks for your comments
We can see chart a little later after run the app.
But I will fix that will show the empty chart even though this table is empty.

**Pull request merged**